manifests/init.pp

```diff
@@ -284,84 +284,119 @@
       $package_name_default      = 'nscd'
       $package_source_default    = undef
       $service_name_default      = 'nscd'
-      case $::operatingsystemmajrelease {
-        '5': {
```

Comment:
Perhaps this could be `'5', '2015', '2016':` to take the multiple versions for Amazon into account. For this to approach to work, we would need to know which dates correspond to EL versions 5, 6 and 7.